.github/workflows/bug_bounty.yml

```
@@ -0,0 +1,62 @@
+name: SalesforceCommerceCloud/pwa-kit/bug_bounty
+on:
+  # For testing - to be removed
+  push:
+    branches:
+      - 'mrt-test-ci-deploy'
+  # Run this job when a new Github release is published
+  release:
+    types: [released]
+
+jobs:
+  deploy-bug-bounty:
+    runs-on: ubuntu-latest
```

nit: Would you consider separating the jobs into two or create a separate GitHub Actions to avoid checking `if: env.IS_LATEST_RELEASE == 'true'` on each step?

```suggestion
jobs:
  check-latest-release:
    runs-on: ubuntu-latest
    ...
  deploy-bug-bounty:
    needs: check-latest-release
    ...
```

---

As suggested, I have separated the latest release check into a separate job.

Here is the flow working (when the condition to trigger is reversed so we proceed if the release is not latest, for testing purposes): https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/13576833436/job/37954917447

And here is the flow failing (when the condition to trigger the flow is correct so that we only deploy on latest):
https://github.com/SalesforceCommerceCloud/pwa-kit/actions/runs/13576920319/job/37955192027

---

This approach also allows us to add other deploy jobs to the workflow later on so this is a really good suggestion @adamraya !